documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -13,10 +13,8 @@ A Kubernetes cluster can host multiple operators, but no more than one per names
 Before installing an operator, ensure that each of these prerequisite requirements is met:
 
 1. [Check environment](#check-environment)
-1. [Download the operator source, if needed](#download-the-operator-source-if-needed)
 1. [Set up the operator Helm chart access](#set-up-the-operator-helm-chart-access)
 1. [Inspect the operator Helm chart](#inspect-the-operator-helm-chart)
```

Comment:
The "Inspect" section needs to be updated so that it discusses using the repo option before it discusses using the file option.